packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -448,7 +448,9 @@ email:
     expires: never
     data_sensitivity:
       - interaction
-  google_oauth_email_first_start:
+    no_lint:
```

Comment:
👀 what's this?

Reply:
glean_parser doesn't like it when every event name in the category starts with the same prefix.  `COMMON_PREFIX` is telling it to ignore that.